### DISCUSSION TITLE
Colocar imagem em um exemplo bootstrap

### DISCUSSION THREAD

**Discussion by jcvilanova** [frontendbr/forum#2268] [Perguntas e Respostas]
Pessoal, peguei como exemplo o código de um modelo bootstrap do tipo bolg e não estou conseguindo inserir minhas imagens no modelo, gostaria de saber como inserir. O código que peguei na página oficial do bootstrap é:

`<div class="p-4 p-md-5 mb-4 text-white rounded bg-dark">
<div class="col-md-6 px-0">
<h1 class="display-4 fst-italic">Title of a longer featured blog post</h1>
<p class="lead my-3">Multiple lines of text that form the lede, informing new readers quickly and efficiently about what’s most interesting in this post’s contents.</p>
<p class="lead mb-0"><a href="#" class="text-white fw-bold">Continue reading...</a></p>
</div>
</div>

<div class="row mb-2">
<div class="col-md-6">
<div class="row g-0 border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative">
<div class="col p-4 d-flex flex-column position-static">
<strong class="d-inline-block mb-2 text-primary">World</strong>
<h3 class="mb-0">Featured post</h3>
<div class="mb-1 text-muted">Nov 12</div>
<p class="card-text mb-auto">This is a wider card with supporting text below as a natural lead-in to additional content.</p>
<a href="#" class="stretched-link">Continue reading</a>
</div>
<div class="col-auto d-none d-lg-block">
<svg class="bd-placeholder-img" width="200" height="250" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Placeholder: Thumbnail" preserveAspectRatio="xMidYMid slice" focusable="false"><title>Placeholder</title><rect width="100%" height="100%" fill="#55595c"/><text x="50%" y="50%" fill="#eceeef" dy=".3em">Thumbnail</text></svg>

</div>
</div>
</div>
<div class="col-md-6">
<div class="row g-0 border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative">
<div class="col p-4 d-flex flex-column position-static">
<strong class="d-inline-block mb-2 text-success">Design</strong>
<h3 class="mb-0">Post title</h3>
<div class="mb-1 text-muted">Nov 11</div>
<p class="mb-auto">This is a wider card with supporting text below as a natural lead-in to additional content.</p>
<a href="#" class="stretched-link">Continue reading</a>
</div>
<div class="col-auto d-none d-lg-block">
<svg class="bd-placeholder-img" width="200" height="250" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Placeholder: Thumbnail" preserveAspectRatio="xMidYMid slice" focusable="false"><title>Placeholder</title><rect width="100%" height="100%" fill="#55595c"/><text x="50%" y="50%" fill="#eceeef" dy=".3em">Thumbnail</text></svg>

</div>
</div>
</div>
</div>`

Eu gostaria de colocar imagens referentes a cada categoria de post que criei, mas quando tento não fica legal.

Gostaria que a imagem tomasse todo esse espaço em cinza em cada uma das divs. Como faço isso ?

**Comment by felquis** [comment]
Olá tudo bem? Qual é seu nome?

Não consigo ver o código que você compartilhou, para compartilhar código você precisa abrir um bloco de código com 3 caracteres assim ``` colocar código aqui ``\` e fechar o bloco

Você pode copiar acima e trocar apenas o conteúdo do meio com seu código, assim conseguiremos tentar te ajudar melhor.

Obrigado.

**Reply by felquis** [reply]
cc @jcvilanova quando puder atualize o código para eu conseguir le-lo e tentar te ajudar.

**Comment by jcvilanova** [comment]
Boa tarde!!!

Irmão, minha dúvida parece ser tão simples que acho que posso explicar rapidinho.
Não estou em casa, meu computador não tá aqui, mas se puder conversar pelo WhatsApp eu posso te explicar rapidinho
Meu número e 98 970220128

Enviado do meu iPhone

Em 23 de abr. de 2022, à(s) 13:22, Felquis G ***@***.***> escreveu:

﻿

cc @jcvilanova<https://github.com/jcvilanova> quando puder atualize o código para eu conseguir le-lo e tentar te ajudar.

**Reply by felquis** [reply]
Não posso responder por outro lugar a não ser pelo fórum.
